H4LCppModule.py

```diff
@@ -19,6 +19,17 @@ def __init__(self):
                 ROOT.gSystem.Load("libPhysicsToolsNanoAODTools.so")
                 ROOT.gROOT.ProcessLine(
                     ".L %s/interface/H4LTools.h" % base)
+        if "/RoccoR_cc.so" not in ROOT.gSystem.GetLibraries():
+            base = "/afs/cern.ch/user/y/yujil/Run3H4l/CMSSW_10_6_20/src/PhysicsTools/NanoAODTools/python/postprocessing/analysis/nanoAOD_vvVBS" 
```

Comment:
Don't add the absolute path. Add the relative path. Use the environment variable `$CMSSW_BASE`